code/data_structures/src/list/singly_linked_list/operations/PalindromeInASinglyLinkedList.c

```diff
@@ -0,0 +1,65 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct node
+{
+    int  info;
+    struct node*  link;
+};
+
+//Function for creating the linked list
+
+    struct node* create(struct node*  start)
+{
+    int proceed = 1;
+    struct node*  temp, *ptr;
+
+    while(proceed == 1)
+    {
+        temp = (struct node *)malloc(sizeof(struct node));
+        temp->link = NULL;
+        puts("Enter info") ;
+        scanf("%d", &temp->info);

```

Comment:
I repeat myself; Please fix the indentation.

Reply:
@iashreya Looks like you have fixed it now. Good job!

Reply:
Thank you!